### ISSUE TITLE
Is offline conversion into HTML a feature of the package?

### ISSUE THREAD

**Issue by srg9000** [tatuylonen/wikitextprocessor#6]
The readme mentions that the package can be used for converting wikitext to HTML, but if I understand correctly, the intended usage of this package is to parse dump files and convert to parse tree objects (WikiNode) which can be later used for tasks such as wikitext to html conversion right?
Thank you

**Comment by tatuylonen** [comment]
It doesn't currently do the conversion all the way to html.  (Some function name is currently a bit misleading in that respect).  The current output is wikitext with templates expanded.  I'll probably implement full HTML output later.  I'm still not sure how exactly mediawiki HTML/CSS formatting works and whether all the information is included in the dumps.

**Comment by srg9000** [comment]
Hi, thanks for your response. I ended up using mediawiki itself. I can't say how mediawiki/parsoid php converts HTML exactly (the way it parses), for templates and modules, it uses the database created from the dump to read the files and expand templates (I separated all templates and modules from the dump and imported them to the database). It doesn't generate css, it adds the style as html tags or inline, and this style information is often in some templates.
